M2/Macaulay2/m2/testing.m2

```diff
@@ -40,7 +40,8 @@ captureTestResult := (desc, teststring, pkg, usermode) -> (
     -- try capturing in the same process
     if isCapturable(teststring, pkg, true) then (
 	checkmsg("capturing", desc);
-	(err, output) := capture(teststring, PackageExports => pkg, UserMode => usermode);
+	-- TODO: adjust and pass argumentMode, instead. This can be done earlier, too.
+	(err, output) := capture(teststring, PackageExports => pkg, UserMode => false);
```

Comment:
Suppose a user gets the source code for two packages that are not part of Macaulay2 and wants to install and test them.  Suppose one depends on the other.  The test will not work if UserMode is false here.  So, I think we have to keep this the way it was.

Reply:
That's not what `UserMode` does for `capture`. It only affects whether the symbols from the test end up in the User dictionary, see #1787. The interpreter inside `capture` uses the same `path` variable as the M2 interpreter that called it, so in your scenario the packages will be loaded fine, as long as `-q` isn't given to M2.

Reply:
But I think you're right that there's an edge case that I had missed. I justpushed a commit to avoid a situation where the command line has `-q` but `UserMode => true` is given, so `capture` isn't used.